run.sh

```diff
@@ -36,16 +36,59 @@ function output() {
   fi
 }
 
+function do_ftrace() {
```

Comment:
let's put this in /hook.sh

it can write a file to keep track of the number of times it has run or maybe last timestamp (date +%s) and run at most every X seconds etc.

Reply:
Done and added file to keep track of max ftrace attempts.